packages/pds/src/db/migrations/20230719T170511044Z-invite-note.ts

```diff
@@ -0,0 +1,12 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('user_account')
+    .addColumn('inviteNote', 'text')
```

Comment:
we use `varchar` everywhere else for text columns (i believe there's no difference under the hood between varchar & text), but I'd like to keep it all `varchar` just for consistencies sake

Reply:
I noticed that but looks like for moderation reports, we're using `text` column so I figured the content for these fields would probably be similar. moving to varchar if there's no difference